examples/tutorials/continuous-monitoring-with-pipeline/monitoring/continuous_monitoring_pipeline.ipynb

```diff
@@ -0,0 +1,746 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.1 Load QnA RAG Evaluation flow as component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eval_qna_rag_metrics_component = load_component(\n",
+    "    \"../../../flows/evaluation/eval-qna-rag-metrics/flow.dag.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.2 Load Perceived Intelligence flow as component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eval_perceived_intelligence_component = load_component(\n",
+    "    \"../../../flows/evaluation/eval-perceived-intelligence/flow.dag.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.3 Load Summarization flow as component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "simple_summarization_component = load_component(\n",
+    "    \"../flows/standard/simple-summarization/flow.dag.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.4 Load Summarization Evaluation flow as component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eval_summarization_component = load_component(\n",
+    "    \"../../../flows/evaluation/eval-summarization/flow.dag.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.3 Load Parquet Converter\n",
+    "The Parquet Converter is a [command component](https://learn.microsoft.com/en-us/azure/machine-learning/reference-yaml-component-command?view=azureml-api-2) that aggregates the results of the evaluations node and logs the metrics to ML Pipelines.  \n",
+    "\n",
+    "![convert_parquet.png](./media/convert_parquet.png)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "convert_parquet_component = load_component(\n",
+    "    \"./components/convert_parquet/convert_parquet.yaml\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3.1 Declare input and output"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_input = Input(\n",
+    "    path=\"./data/monitoring_dataset.jsonl\",\n",
+    "    type=AssetTypes.URI_FILE,\n",
+    "    mode=\"mount\",\n",
+    ")\n",
+    "\n",
+    "eval_results_output = Output(\n",
+    "    # Provide custom flow output file path if needed\n",
+    "    path=\"azureml://datastores/<data_store_name>/paths/<path>\",\n",
```

Comment:
Thanks for contributing such a realistic and complete scenario! I saw you created a CI for this sample, to make the CI pass, the path on datastore should be a real one? 
Since this sample includes scenarios like schedule & pipeline endpoint, not sure if it'd be too heavy to run in CI.

Reply:
For your case, maybe put it in [integration sample folder](https://github.com/microsoft/promptflow/tree/main/examples/flows/integrations) would be better.

Reply:
> Thanks for contributing such a realistic and complete scenario! I saw you created a CI for this sample, to make the CI pass, the path on datastore should be a real one? Since this sample includes scenarios like schedule & pipeline endpoint, not sure if it'd be too heavy to run in CI.

Thank you for your feedback !

I reuse some steps from the [Run flows in Azure ML pipeline](https://github.com/microsoft/promptflow/blob/main/examples/tutorials/run-flow-with-pipeline/pipeline.ipynb) tutorial so I assumed they might also work.
Would you recommend to remove the schedule & pipeline endpoint steps ?

Reply:
agreed with Han, as this example contains many components inside Azure ML, making it with a workflow CI may increase the maintain effort - maybe we can just put it inside integration folder (as Han shared above) and remove those workflow file

Reply:
Move to the [integration sample folder](https://github.com/microsoft/promptflow/tree/main/examples/flows/integrations) is a good idea.

And by the way please manual delete the generated pipeline, and re-run the readme.py until it blames no more.

Then it will be ok.

Reply:
> > Thanks for contributing such a realistic and complete scenario! I saw you created a CI for this sample, to make the CI pass, the path on datastore should be a real one? Since this sample includes scenarios like schedule & pipeline endpoint, not sure if it'd be too heavy to run in CI.
> 
> Thank you for your feedback !
> 
> I reuse some steps from the [Run flows in Azure ML pipeline](https://github.com/microsoft/promptflow/blob/main/examples/tutorials/run-flow-with-pipeline/pipeline.ipynb) tutorial so I assumed they might also work. Would you recommend to remove the schedule & pipeline endpoint steps ?

The item you mentioned is also served as a gated test for pf releasing.

So these folders doesn't accept new changes.

Reply:
I have moved the files to the integration folder.